build.sbt

```diff
@@ -91,7 +91,8 @@ lazy val tests = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-testkit" % V.cats % "test",
       "org.typelevel" %%% "cats-laws"    % V.cats % "test",
-      "org.typelevel" %%% "algebra-laws" % V.algebra % "test"
+      "org.typelevel" %%% "algebra-laws" % V.algebra % "test",
+      "org.typelevel" %% "cats-testkit-scalatest" % V.catsTestkit % "test"
```

Comment:
I'm a little surprised why this works, because I thought a `%%%` is required here. :thinking:

Reply:
Wow, i was looking everywhere for some kind of a mistake due to NoClassDefFound but missed that one... Thanks for pointing it out :) I will add the `%%%` and revert changes from BitSetTest - it is working just fine.